ros2bag/ros2bag/verb/record.py

```diff
@@ -114,30 +114,7 @@ def main(self, *, args):  # noqa: D102
             except (InvalidQoSProfileException, ValueError) as e:
                 return print_error(str(e))
 
```

Comment:
Nit: add comment assuming that both are not true given the check on line 94.

Reply:
Was able to get rid of the if condition above record call entirely by placing the check at the top of the function.